app/resources/css-default.template

```diff
@@ -204,20 +204,34 @@ select {
 /* Expand Forms */
 
 .expanded-part {
-  background: #FFF9C4;
-  padding: 5px;
-  border: 1px solid gray;
-  margin-left: auto;
+  padding: 8px;
+  margin-left: 5px;
   margin-right: auto;
```

Comment:
Do you know what it means by setting fixed pixels for margin-left and "auto" for margin-right?
(I know that setting "auto" for both means centering.)
Maybe it makes more sense to set both to 5px?

Also, do you have any intention to specify padding and margin separately? It may be more straight-forward to specify just one of them e.g., set margin-left and margin-right to 13px, margin-top and margin-bottom to 8px, no padding.